tools/ci_build/github/azure-pipelines/templates/windowsai-nuget-build.yml

```diff
@@ -100,7 +100,13 @@ steps:
         searchFolder: '$(Build.BinariesDirectory)'
         testRunTitle: 'Unit Test Run'
       condition: succeededOrFailed()
-
+  - ${{ if and(eq(parameters.BuildArch, 'x64'), eq(parameters.BuildForStore, 'false')) }}:
+    - task: PublishPipelineArtifact@0
+      displayName: 'Publish Pipeline Artifact: downlevel-test-artifact'
+      inputs:
+        artifactName: downlevel-test-artifact
+        targetPath: '$(Build.BinariesDirectory)\RelWithDebInfo\RelWithDebInfo'
```

Comment:
The folder is very very big. We'll run out of disk space quickly. We already hit such issues on aiinfra.visualstudio.com. Could you only take the PE files and test data out?

Reply:
Changed to just publish the test binaries and collateral needed. Its now just 77MB.